enterprise/security/src/main/java/org/neo4j/server/security/enterprise/auth/ShiroAuthManager.java

```diff
@@ -81,6 +83,7 @@ public void init() throws Throwable
     {
         super.init();
 
+        roleRepository.init();
```

Comment:
I think you should use the external lifecycle to handle the various init/start/stop/shutdown of components and not have your own internal mini-lifecycle